content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,39 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+This data may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements. e.g. A
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly complex.
+
+DVC allows a single directory to contain all your project artifacts. The
+workspace is the directory containing _user visible_ part of your
+<abbr>project</abbr> e.g. raw datasets, source code, ML models, etc. Users work
+in this directory using their data and model files that and manipulate the
+contents through DVC commands.
+
+Files and directories in the workspace can be added to DVC (`dvc add`) or they
+can be downloaded from external sources (`dvc get`, `dvc import`,
+`dvc import-url`). Changes to the files, directories, notebooks, models, and in
+general machine learning file system can be tracked (`dvc commit`) and versioned
+in Git (`dvc checkout`). They can be removed (`dvc remove`) from the workspace.
```

Comment:
- It sounds like we're saying that `dvc checkout` versions data with Git. Can you clarify a bit?
- "They can be removed" -> "Tracked data can be..."

Reply:
Modified in 6cc0f6c

Reply:
> data, notebooks, models, and any related machine learning artifact

That list got a little long? Prob no need for "machine learning" here since the term is found elsewhere in the doc...

> their content can be synchronized (`dvc checkout`)

Good one! But needs some clarifying (sync what)?

Also, people usually need checkout before commit (which is included in add/repro). So maybe something like:

"When switching Git revisions in your |repo|, you can use `dvc checkout` to sync |metafiles| with the other workspace files and directories. If you manually modify tracked data, use `dvc commit` to record the changes with DVC. And if needed, tracked data can be removed..."

Reply:
> "When switching between |repository| versions, use dvc checkout to sync DVC-tracked data with Git-tracked |metafiles|. If you manually modify the workspace status, use dvc commit to record the changes. And if needed, tracked data can be removed..."

This sentence looks more suitable to use cases or the user guide. Here I would like to mention just the capabilities. Conceptually `dvc checkout` is similar to `git checkout` and `dvc commit` is akin to. `git commit`. This can be turned into a bullet list maybe,

- You can add your files and directories ...
- You can download external sources
- You can synchronize your workspace with the cache
- You can do this
- You can do that

I feel this paragraph is too dense.

Reply:
But reading again, this looked rather like a political ad than a technical documentation. Maybe "you can"s are spurious. Just 

- Add your files (`dvc add`)
- Download external sources (`dvc get`...)
- ...

is better.